packages/sku/src/services/vite/helpers/html/createPreRenderedHtml.ts

```diff
@@ -28,6 +28,7 @@ export function createPreRenderedHtml({
       clientContext = await render.provideClientContext({
         site,
         url,
+        environment: process.env.NODE_ENV,
```

Comment:
This isn't where `environment` should come from. This environment is actually the [sku environment](https://seek-oss.github.io/sku/#/./docs/configuration?id=environments) the page is being rendered in. I don't think our vite implementation supports environments just yet, so would remove this until we have proper support for pre-rendering multiple environments.